lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -72,19 +93,9 @@ public function getTests($name, $legacyTests = false)
             $tests[] = array(str_replace($fixturesDir.'/', '', $file), $message, $condition, $templates, $exception, $outputs);
         }
 
-        if (!$tests) {
-            // add a dummy test to avoid a PHPUnit message
-            return array(array('not', '-', '', array(), '', array()));
-        }
```

Comment:
This should go because if there are no tests, it should fail in the same way unit tests fail if there are no test cases
```
(1) Warning
No tests found in class "FooBarTest".
```